go/vt/mysqlctl/tmutils/schema.go

```diff
@@ -63,94 +63,129 @@ func (tds TableDefinitions) Swap(i, j int) {
 	tds[i], tds[j] = tds[j], tds[i]
 }
 
-// FilterTables returns a copy which includes only whitelisted tables
+type TableFilter struct {
```

Comment:
nit: a doc comment will be nice.

Reply:
there's already docs in both NewTableFilter and FilterTables, so I felt it was redundant. if you insist, i will copy the content.

Reply:
no need to duplicate.

Reply:
Precommit didn't catch this? All exported names require a comment.
If it's not useful outside this package, it's better to make it unexported.
Same with NewTableFilter.

Reply:
I'll add comment then. This is intended to actually be used outside in the `mysqlctl` package, and replace FilterTables, in a separate PR.